client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -354,6 +364,47 @@ const PluginsBrowser = ( {
 	);
 };
 
+const SearchHeader = ( props ) => {
+	const { doSearch, search, siteSlug, title, searchBoxRef } = props;
+
+	return (
+		<div className="plugins-browser__searchbox-container">
+			<div className="plugins-browser__searchbox-header">{ title }</div>
+			<div className="plugins-browser__searchbox" ref={ searchBoxRef }>
+				<SearchBox doSearch={ doSearch } search={ search } />
+			</div>
+			<PopularSearches
+				siteSlug={ siteSlug }
+				searchTerms={ [ 'shipping', 'seo', 'portfolio', 'chat', 'mailchimp' ] }
+			/>
+		</div>
+	);
+};
+
+const PopularSearches = ( props ) => {
+	const { searchTerms, siteSlug } = props;
+	const translate = useTranslate();
+
+	return (
+		<div className="plugins-browser__recommended-searches">
+			<div className="plugins-browser__recommended-searches-title">
+				{ translate( 'Most popular searches' ) }
+			</div>
+
+			<div className="plugins-browser__recommended-searches-list">
+				{ searchTerms.map( ( searchTerm ) => (
+					<a
+						href={ `/plugins/${ siteSlug || '' }?s=${ searchTerm }` }
+						className="plugins-browser__recommended-searches-list-item"
+					>
+						{ searchTerm }
+					</a>
+				) ) }
+			</div>
+		</div>
+	);
+};
+
```

Comment:
Can we move new components under `plugins` for the `SearchHeader`? This file has already become too big and hard to handle.

Reply:
Bundling their respective CSS and removing it from `client/my-sites/plugins/plugins-browser/style.scss` too.

Reply:
sure, was thinking on doing that before pushing the PR to review